### ISSUE TITLE
Allow `range` accept only one argument 

### ISSUE THREAD

**Issue by martinsik** [ReactiveX/rxjs#4332]
## Feature Request

**Is your feature request related to a problem? Please describe.**
I'm never able to remember whether RxJS `range` has a variant that accepts only one argument. For example in lodash I can use `range(10)` and it'll generate an array of 10 items. However, in RxJS I always have to use two parameters `range(0, 10)`.

```
import { range } from 'rxjs';
import _ from 'lodash';

// will work
range(0, 10).subscribe(console.log)

// won't work
range(10).subscribe(console.log)

// will work
console.log(_.range(10))
```

https://stackblitz.com/edit/rxjs6-demo-n1wwfd?file=index.ts

This isn't consistent across different Rx implementations:

RxPY does allow only one param https://github.com/ReactiveX/RxPY/blob/master/rx/operators/observable/range.py#L6.
RxJava doesn't https://github.com/ReactiveX/RxJava/blob/2.x/src/main/java/io/reactivex/Observable.java#L3540
RxSwift does ... something https://github.com/ReactiveX/RxSwift/blob/master/RxSwift/Observables/Range.swift#L30
Lodash does https://lodash.com/docs/4.17.11#range

**Describe the solution you'd like**

Ideally, if I use just one argument it'll be used as `count` parameter  and `start` will be set to `0`.

So using `range(10)` and `range(0, 10)` will produce the same result.

**Describe alternatives you've considered**

I can keep on using `range(0, 10)` but if I could use just `range(10)` it would be easier to remember and probably shouldn't break any existing functionality.

**Comment by arpadvas** [comment]
I can have a look into it and create a PR...

**Comment by martinsik** [comment]
This feature was already merged in #4360 so I'm closing this issue.
